sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1503,16 +1488,21 @@ def _post_port_sfp_info_and_dom_thr_to_db_once(self, port_mapping, xcvr_table_he
                 continue
             rc = post_port_sfp_info_to_db(logical_port_name, port_mapping, xcvr_table_helper.get_intf_tbl(asic_index), transceiver_dict, stop_event)
             if rc != SFP_EEPROM_NOT_READY:
-                self.dom_db_utils.post_port_dom_thresholds_to_db(logical_port_name)
-                # Read the VDM thresholds and post them to the DB
-                self.vdm_db_utils.post_port_vdm_thresholds_to_db(logical_port_name)
-
-                # Do not notify media settings during warm reboot to avoid dataplane traffic impact
                 if is_warm_start == False:
                     media_settings_parser.notify_media_setting(logical_port_name, transceiver_dict, xcvr_table_helper, port_mapping)
-                    transceiver_dict.clear()
             else:
                 retry_eeprom_set.add(logical_port_name)
+        
+        dom_thresholds_cache = {}
```

Comment:
These new lines should also be part of the "if" section of line 1490? "if rc != SFP_EEPROM_NOT_READY:"

Reply:
No, we added check:
```python
if logical_port_name not in retry_eeprom_set:
```